virtool/tasks/client.py

```diff
@@ -1,10 +1,15 @@
+import asyncio
 import logging
 from abc import ABC, abstractmethod
 from typing import List
 
-from aioredis import Redis
+from aioredis import (
+    Redis,
+    ConnectionClosedError,
+)
 
 REDIS_TASKS_LIST_KEY = "tasks"
+logging.getLogger("tasksClient")
```

Comment:
I'd like to keep these more general and limited in number. There were too many logger names before and it made it hard to scan the logs visually.
```suggestion
logging.getLogger("tasks")
```